huaweicloud/services/acceptance/acceptance.go

```diff
@@ -1458,10 +1459,17 @@ func TestAccPreCheckLTSCrossAccountAccess(t *testing.T) {
 
 // lintignore:AT003
 func TestAccPreCheckLTSCCEAccess(t *testing.T) {
-	if HW_LTS_LOG_STREAM_ID == "" || HW_LTS_LOG_GROUP_ID == "" ||
-		HW_LTS_CCE_CLUSTER_ID == "" || HW_LTS_CCE_HOST_GROUP_ID == "" {
-		t.Skip("The cce access config of HW_LTS_LOG_STREAM_ID, HW_LTS_LOG_GROUP_ID, HW_LTS_CCE_CLUSTER_ID" +
-			" and HW_LTS_CCE_HOST_GROUP_ID must be set for the acceptance test")
+	if HW_LTS_CCE_CLUSTER_ID == "" {
+		t.Skip("The cce access config of HW_LTS_CCE_CLUSTER_ID must be set for the acceptance test")
+	}
+}
+
+// lintignore:AT003
+func TestAccPreCheckLTSHostGroup(t *testing.T) {
+	// LTS host groups support updating associated hosts, so creation and updating each require a host.
+	hostIds := strings.Split(HW_LTS_HOST_IDS, ",")
+	if len(hostIds) < 2 {
+		t.Skip("The length of HW_LTS_HOST_IDS must be 2 for the host group acceptance test")
```

Comment:
please confirm this description, the logic means the length must be greater than 1.

Reply:
done